### ISSUE TITLE
Report an error when item is empty

### ISSUE THREAD

**Issue by ShibaInu0413** [MikaelPorttila/rss#49]
When I tested, I used an rss with an empty item, and his error was as follows

```
TypeError: Cannot read properties of undefined (reading 'map')
    at mapRss2ToFeed (https://deno.land/x/rss@0.5.8/src/mappers/mapper.ts:245:26)
    at toFeed (https://deno.land/x/rss@0.5.8/src/mappers/mapper.ts:33:14)
    at parseFeed (https://deno.land/x/rss@0.5.8/src/deserializer.ts:36:10)
    at eventLoopTick (ext:core/01_core.js:181:11)
    at async Server.handler (file:///src/main.ts:16:23)
    at async Server.#respond (https://deno.land/std@0.177.0/http/server.ts:299:18)
```

**Comment by MikaelPorttila** [comment]
Hi, thank you for reporting on this issue and your PR 🤝, your changes are included in the **0.6.0** release. 
I'll extend our tests so we cover this issue for all the feed types.

BR
Micke

**Comment by ShibaInu0413** [comment]
No, I'm happy to contribute to this package
